misc/python/materialize/checks/common.py

```diff
@@ -0,0 +1,26 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+SAMPLE_KAFKA_SCHEMA = """
```

Comment:
I think `SIMPLE` would be more appropriate in the name.

Reply:
Maybe even better: `KAFKA_SCHEMA_WITH_SINGLE_STRING_FIELD` (or similar)